Content.Server/Fluids/EntitySystems/PuddleSystem.Evaporation.cs

```diff
@@ -46,6 +46,7 @@ private void TickEvaporation()
             if (!_solutionContainerSystem.ResolveSolution(uid, puddle.SolutionName, ref puddle.Solution, out var puddleSolution))
                 continue;
 
+            // Yes, this means that 50u water + 50u holy water evaporates twice as fast as 100u water.
```

Comment:
you should either make an issue or fix it :3

Reply:
Yeah mulling over how to fix it (probably just do average evaporation speed), and will probably get it as part of making it predicted.

~~Physically accurate would make it proportional to exposed surface area tho.~~